css/bootstrap.scss

```diff
@@ -1503,110 +1506,6 @@ body.no-scroll {
   width: 100%;
 }
```

Comment:
Do we still need these `no-scroll` styles? From my brief `ctrl-f` it looks like they're not being used anymore.